src/styles/cards/cards_blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+@use "./cards_utils/__variables" as *;
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    align-self: center;
+  }
+
+  &__lable {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #{$main-color};
+    padding-top: 40px;
+  }
+
+  &__code {
+    font-weight: 400;
```

Comment:
It's the default value
![image](https://user-images.githubusercontent.com/91411692/222675732-2c874b59-93fa-4eae-9add-f57c0ce9ebcb.png)

![image](https://user-images.githubusercontent.com/91411692/222675648-684def54-5cf1-4c04-b707-21d0503a2dfb.png)

```suggestion
```